.ci-operator.yaml

```diff
@@ -1,5 +1,4 @@
-build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: rhel-8-release-golang-1.20-openshift-4.14
+build_root_image:
+  name: release
+  namespace: openshift
+  tag: rhel-8-release-golang-1.20-openshift-4.14
```

Comment:
I think we want to change this to use go 1.21, no?

Reply:
This is the existing config from openshift/release.
a 1.21 config seems to break with when we are using 1.20 in the repo.